### PR TITLE
Image

### PR DESCRIPTION
**Pull request by malgorzata-bielak**

### CODE REVIEW

File: src/components/EventItem.js [hunk 1]
```diff
@@ -9,9 +9,16 @@ const EventItem = ({ title, image, startDate, endDate, organisator, city, catego
 
   return (
     <Link to={`edit/${id}`}>
-      <div>{image}</div>
+      {image ? (
+        <div>
```
Comment (mateusz-bielak) [inline src/components/EventItem.js:13]: You don't need that `div` I believe :)

File: src/containers/EditEventPage.js [hunk 2]
```diff
@@ -20,8 +21,14 @@ class EditEventPage extends React.Component {
   render() {
     return (
       <>
-        <EventForm event={this.props.event} onSubmit={this.onSubmit} />
-        <button onClick={this.onRemoveClick}>Remove event</button>
+        {this.props.event ? (
+          <>
```
Comment (mateusz-bielak) [inline src/containers/EditEventPage.js:25]: And you probably don't need these inner fragments too 🤔

File: src/containers/EditEventPage.js [hunk 3]
```diff
@@ -20,8 +21,14 @@ class EditEventPage extends React.Component {
...
+            <EventForm event={this.props.event} onSubmit={this.onSubmit} />
+            <button onClick={this.onRemoveClick}>Remove event</button>{" "}
```
Comment (mateusz-bielak) [inline src/containers/EditEventPage.js:27]: `{" "}` ? 🤔

**Review by mateusz-bielak — APPROVED** [review]

File: src/containers/EditEventPage.js (hunk 2 above)
Comment (malgorzata-bielak) [reply src/containers/EditEventPage.js:25]: I need this one ;)

Comment (mateusz-bielak) [reply src/containers/EditEventPage.js:25]: Well, then maybe you don't need the external one :)

File: src/containers/EditEventPage.js (hunk 3 above)
Comment (malgorzata-bielak) [reply src/containers/EditEventPage.js:27]: I don't know why but sometimes it is added automatically by vsc

**Pull request merged**